docs/source/locale/ja/LC_MESSAGES/markdown.po

```diff
@@ -1108,14 +1108,14 @@ msgid ""
 "container or Kubernetes workloads must run inside a systemd unit. After a"
 " successful update of an image, the containers using the image get "
 "updated by restarting the systemd units they run in. Please refer to "
-"`quadlet(5)` on how to run Podman under systemd."
+"`podman-systemd.unit(5)` on how to run Podman under systemd."
```

Comment:
Are you sure we should be changing the Japanese translation here?

Reply:
That's based on a recursive grep over the docs tree; it's the only place in the translations where "`quadlet(5)`" previously existed. I changed it for consistency, purely on the speculation that it might matter.

What this does to the translations or even why it was present there in the first place is a question for the Japanese translator.

My outsider perspective is that this is a null translation, there being no Kanji in sight, but don't listen to me. I know next to nothing about i18n things.

Reply:
I would not worry about that at all, nobody seems to actually maintain the translation. I don't know why this was ever accepted in the first place without any docs on what a maintainer should do with that and who is supposed to update it.